packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-pwd-recovery-mfa/java/integrationsteps.md

```diff
@@ -1,3 +1,98 @@
+
+1. `AuthenticationStatus` = `AWAITING_AUTHENTICATOR_SELECTION` <br>
+     This status indicates that there are required authenticators that need to be verified.
+
+2. `Authenticators` = List of [authenticators](https://github.com/okta/okta-idx-java/blob/master/api/src/main/java/com/okta/idx/sdk/api/client/Authenticator.java) to be verified (in this case, there is only the email authenticator). <br>
+    Authenticators are the factor credentials that are owned or controlled by the user. These are verified during authentication.
+
+After receiving the `AWAITING_AUTHENTICATOR_SELECTION` status and the list of authenticators to be verified, provide the user with a form to select the authenticator to verify. In other words, provide the user with a list of recovery factors to use. In this use case, email is the only recovery factor available:
+
+<div class="common-image-format">
+
+![Displays the recovery factor list to use (only email for this use case)](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-pwd-recovery-screenshot-choose-auth-java.png)
+
+</div>
+
+### Step 3: User selects email authenticator
+
+The user selects **Email** as the authenticator to recover their password. Pass the selected authenticator to the `IDXAuthenticationWrapper.selectAuthenticator()` method:
+
+```java
+authenticationResponse = idxAuthenticationWrapper.selectAuthenticator(proceedContext, authenticator);
+```
+
+This Java SDK method sends the email authenticator selection to Okta. Okta sends a code to the user's email and the Java SDK returns `AuthenticationStatus=AWAITING_AUTHENTICATOR_VERIFICATION` to your client app. This status indicates that the authentication flow is waiting for an authenticator verification, in this case, an email verification code. You need to build a form to capture the code from the user.
+
+<div class="common-image-format">
+
+![Displays the email verification code input form](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-pwd-recovery-verify-code-java.png)
+
+</div>
+
+> **Note:** The email sent to the user has a **Reset Password** link that isn't yet supported. Use the provided code instead. See [Limitations: Passwordless sign-in with magic links](/docs/guides/oie-embedded-sdk-limitations/main/#passwordless-sign-in-with-magic-links).
+
+### Step 4: User submits email verification code
+
+The user receives the verification code in their email and submits it through the verify code form. Use [`VerifyAuthenticationOptions`](https://github.com/okta/okta-idx-java/blob/master/api/src/main/java/com/okta/idx/sdk/api/model/VerifyAuthenticatorOptions.java) to capture the code and send it to the `IDXAuthenticationWrapper.verifyAuthenticator()` method:
```

Comment:
Nitpick, but shouldn't "verify code" be "**Verify Code**", given that this exact term is used in the example UI form?

Reply:
ok, I will change it to "Verify Code". I wanted to use a generic common description of the form since "Verify Code form" is just an example in this use case, other developers will probably use another title.

Reply:
Ah, I get your point here. TBH, I think either would be OK. This one can be resolved, but feel free to change it back if you'd prefer the original.